urg.repos

```diff
@@ -0,0 +1,25 @@
+repositories:
+  diagnostics:
```

Comment:
It seems to me like `diagnostics`, `ros2_console`, and `ros2_time` are no longer required to compile the urg stuff.  At least, I didn't need them to compile.  If that's the case, please remove the from this repos file.